### DISCUSSION TITLE
Suggestion to add Kolors Virtual Space

### DISCUSSION THREAD

**Discussion by ALBIHANY** [lllyasviel/stable-diffusion-webui-forge#1643] [Ideas]
Any chance to add this amazing space? 

https://huggingface.co/spaces/Kwai-Kolors/Kolors-Virtual-Try-On

**Comment by danilomaiaweb** [comment]
very nice

**Comment by ishadowx27** [comment]
I am also waiting for this. I think it is probably not as good as flux in terms of quality but for people with way less VRAM it could be very useful.

**Comment by wangyanbn** [comment]
this space is calling the cloud api,  no opensouced model.
